dotnet/xml/ns-Microsoft.Extensions.ML.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.Extensions.ML">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Utilities for extending the ML.NET API, including the prediction engine pool</summary>
     <remarks>To be added.</remarks>
```

Comment:
What about the remarks? Is it a mandatory field? Will the "To be added" text show up in the final docs somewhere? If so, can we either add valid remarks or remove the field?

Reply:
It only shows up if you customize the default text. Otherwise the element is not added. Thanks for checking